### PR TITLE
2.0.0-release

### PR DESCRIPTION
**Pull request by antoinesd**
update js files for renaming and release
create blog post

### CODE REVIEW

File: build.js [hunk 1]
```diff
@@ -135,13 +135,13 @@ function build() {
...
-      '/download/keycloak-swarm': 'http://repo2.maven.org/maven2/org/wildfly/swarm/servers/keycloak/' + versions.CURRENT_RELEASE + '/keycloak-' + versions.CURRENT_RELEASE + '-swarm.jar',
-      '/download/management-console-swarm': 'http://repo2.maven.org/maven2/org/wildfly/swarm/servers/management-console/' + versions.CURRENT_RELEASE + '/management-console-' + versions.CURRENT_RELEASE + '-swarm.jar',
-      '/download/swagger-ui-swarm': 'http://repo2.maven.org/maven2/org/wildfly/swarm/servers/swagger-ui/' + versions.CURRENT_RELEASE + '/swagger-ui-' + versions.CURRENT_RELEASE + '-swarm.jar',
-      '/download/web-swarm': 'http://repo2.maven.org/maven2/org/wildfly/swarm/servers/web/' + versions.CURRENT_RELEASE + '/web-' + versions.CURRENT_RELEASE + '-hollowswarm.jar',
+      '/download/swarmtool': 'http://repo2.maven.org/maven2/io/thorntail/swarmtool/'+ versions.CURRENT_RELEASE + '/swarmtool-' + versions.CURRENT_RELEASE + '-standalone.jar',
+      '/download/microprofile-hollowswarm': 'http://repo2.maven.org/maven2/io/thorntail/servers/microprofile/' + versions.CURRENT_RELEASE + '/microprofile-' + versions.CURRENT_RELEASE + '-hollowswarm.jar',
```
Comment (Ladicek) [inline build.js]: This line is wrong, it's named `-hollow-thorntail`.

File: build.js [hunk 2]
```diff
@@ -135,13 +135,13 @@ function build() {
...
+      '/download/microprofile-jpa-hollowswarm': 'http://repo2.maven.org/maven2/io/thorntail/servers/microprofile-jpa/' + versions.CURRENT_RELEASE + '/microprofile-jpa-' + versions.CURRENT_RELEASE + '-hollows-thorntail.jar',
```
Comment (Ladicek) [inline build.js]: This line is wrong, it's named `-hollow-thorntail`.

File: build.js [hunk 3]
```diff
@@ -154,7 +154,8 @@ function build() {
       '/docs/2018-3-3': 'http://docs.wildfly-swarm.io/2018.3.3',
       '/docs/2018-4-1': 'http://docs.wildfly-swarm.io/2018.4.1',
       '/docs/2018-5-0': 'http://docs.wildfly-swarm.io/2018.5.0',
-      '/docs/HEAD': 'http://docs.wildfly-swarm.io/2018.6.0-SNAPSHOT/',
+      '/docs/2018-5-0': 'http://docs.wildfly-swarm.io/2.0.0.Final',
```
Comment (Ladicek) [inline build.js]: Should be `/docs/2.0.0.Final`?

File: build.js [hunk 4]
```diff
@@ -135,13 +135,13 @@ function build() {
...
+      '/download/microprofile-jpa-hollowswarm': 'http://repo2.maven.org/maven2/io/thorntail/servers/microprofile-jpa/' + versions.CURRENT_RELEASE + '/microprofile-jpa-' + versions.CURRENT_RELEASE + '-hollows-thorntail.jar',
+      '/download/keycloak-swarm': 'http://repo2.maven.org/maven2/io/thorntail/servers/keycloak/' + versions.CURRENT_RELEASE + '/keycloak-' + versions.CURRENT_RELEASE + '-thorntail.jar',
+      '/download/management-console-swarm': 'http://repo2.maven.org/maven2/io/thorntail/servers/management-console/' + versions.CURRENT_RELEASE + '/management-console-' + versions.CURRENT_RELEASE + '-thorntail.jar',
+      '/download/swagger-ui-swarm': 'http://repo2.maven.org/maven2/io/thorntail/servers/swagger-ui/' + versions.CURRENT_RELEASE + '/swagger-ui-' + versions.CURRENT_RELEASE + '-thorntail.jar',
+      '/download/web-thorntail': 'http://repo2.maven.org/maven2/io/thorntail/servers/web/' + versions.CURRENT_RELEASE + '/web-' + versions.CURRENT_RELEASE + '-hollow-thorntail.jar',
```
Comment (Ladicek) [inline build.js]: Not sure if we can change the redirect like this. I don't know where the redirects are used, but at least we should keep the old redirects for a while, until we find and change all the usages.

File: src/posts/announce-2.0.0.Final.adoc [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+---
+
+== Thorntail 2.0.0.Final
+
+The new name of WildFly Swarm is _Thorntail_! Enjoy the freedom of flying high like a Thorntail bird!
+
+=== What's New?
+
+* Did we say we have a new name: Thorntail?
+* We give up the `YYYY.MM.VV` versioning scheme and switched to a more convenient `major.minor.micro` versioning
```
Comment (Ladicek) [inline src/posts/announce-2.0.0.Final.adoc]: If "switched to", then should be "gave up"?

File: src/posts/announce-2.0.0.Final.adoc [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+** Thorntail MicroProfile REST Client implementation introduces a non-portable feature - CDI Interceptors Support, see docs http://docs.wildfly-swarm.io/2.0.0.Final/#_cdi_interceptors_support[for more information and examples]
+
+++++
+<!-- more -->
+++++
+
+==== Migration Notes
+
+Migration from WildFly Swarm 2018.5.0 to Thorntail 2.0.0.Final is straightforward.
+Packages and class names have not changed.
```
Comment (Ladicek) [inline src/posts/announce-2.0.0.Final.adoc]: I'd also mention that configuration keys didn't change. And also the set of artifacts didn't change, only how to locate them.

File: src/posts/announce-2.0.0.Final.adoc [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+Therefore, you will most likely only need to change the project build configuration.
+
+==== TL;DR
+
+1. `groupId` of all artifacts now starts with `io.thorntail` prefix, replacing the previous value of `org.wildfly.swarm`
+2. The Maven plugin also has a new `artifactId`: `thorntail-maven-plugin`
+3. The suffix of a generated uberjar has changed to `-thorntail.jar`,
```
Comment (Ladicek) [inline src/posts/announce-2.0.0.Final.adoc]: Also hollow JAR suffix changed.

**Comment by Ladicek** [comment]
I also think we should mention in the migration section that Fabric8 Maven plugin users need to upgrade to 3.5.40, which has the `thorntail-v2` generator and health check enricher (in addition to the `wildfly-swarm` generator / enricher that has been present for a long time and has the same feature set).

File: src/posts/announce-2.0.0.Final.adoc (hunk 7 above)
Comment (mkouba) [reply src/posts/announce-2.0.0.Final.adoc]: Right!

File: src/posts/announce-2.0.0.Final.adoc (hunk 5 above)
Comment (mkouba) [reply src/posts/announce-2.0.0.Final.adoc]: What about "We have left"...?

File: src/posts/announce-2.0.0.Final.adoc (hunk 6 above)
Comment (mkouba) [reply src/posts/announce-2.0.0.Final.adoc]: Good points.

**Comment by mkouba** [comment]
@Ladicek  Could you pls add something about the FMP 3.5.40?

**Comment by Ladicek** [comment]
Ah I can push to Antoine's branch! OK, I'll try.

File: build.js (hunk 4 above)
Comment (antoinesd) [reply build.js]: agree: I'll let the old ones and add new ones below

**Comment by Ladicek** [comment]
Added a commit with FMP migration instructions, @mkouba could you please take a look? At least formally :-)

**Comment by mkouba** [comment]
@Ladicek Looks good.

**Review by Ladicek — APPROVED** [review]

**Review by mkouba — APPROVED** [review]

File: build.js [hunk 8]
```diff
@@ -135,13 +135,19 @@ function build() {
...
-      '/download/keycloak-swarm': 'http://repo2.maven.org/maven2/org/wildfly/swarm/servers/keycloak/' + versions.CURRENT_RELEASE + '/keycloak-' + versions.CURRENT_RELEASE + '-swarm.jar',
-      '/download/management-console-swarm': 'http://repo2.maven.org/maven2/org/wildfly/swarm/servers/management-console/' + versions.CURRENT_RELEASE + '/management-console-' + versions.CURRENT_RELEASE + '-swarm.jar',
-      '/download/swagger-ui-swarm': 'http://repo2.maven.org/maven2/org/wildfly/swarm/servers/swagger-ui/' + versions.CURRENT_RELEASE + '/swagger-ui-' + versions.CURRENT_RELEASE + '-swarm.jar',
-      '/download/web-swarm': 'http://repo2.maven.org/maven2/org/wildfly/swarm/servers/web/' + versions.CURRENT_RELEASE + '/web-' + versions.CURRENT_RELEASE + '-hollowswarm.jar',
+      '/download/swarmtool': 'http://repo2.maven.org/maven2/io/thorntail/swarmtool/'+ versions.CURRENT_RELEASE + '/swarmtool-' + versions.CURRENT_RELEASE + '-standalone.jar',
+      '/download/microprofile-hollow-thorntail': 'http://repo2.maven.org/maven2/io/thorntail/servers/microprofile/' + versions.CURRENT_RELEASE + '/microprofile-' + versions.CURRENT_RELEASE + '-hollow-thorntail.jar',
```
Comment (kenfinnigan) [inline build.js:139]: Should the "-swarm" ones be removed?

**Comment by kenfinnigan** [comment]
I think src/downloads.adoc needs updating to new -thorntail hollow jar links and src/index.adoc to refer to new Thorntail BOM

File: build.js (hunk 8 above)
Comment (Ladicek) [reply build.js:139]: I think we kept them to make sure old links/bookmarks still work. If the only links are from `src/downloads.adoc` and `src/index.adoc`, then we can remove them.

**Comment by Ladicek** [comment]
@kenfinnigan Updated `src/index.adoc` and `src/download.adoc` per your comments above. This is ready for merge IMHO. Reviewing the last commit would be nice tho :-)

**Comment by Ladicek** [comment]
(Obviously there's still a lot of places that refer to WildFly Swarm. I didn't try to hunt them all down. Some of them, e.g. historical blog posts, shouldn't be changed IMHO.)

**Review by kenfinnigan — APPROVED** [review]

**Pull request merged**